scripts/data_collection/README.md

```diff
@@ -0,0 +1,62 @@
+# Data collection tools
+
+## `feedback_aggregator.py`
```

Comment:
Maybe rename to user_data_aggregator?

Reply:
maybe, ok, will do it

Reply:
or other better name, as it is not just about feedback

Reply:
better @onmete ?